client/src/LocalStorage/LocalStorageInit.js

```diff
@@ -0,0 +1,7 @@
+const LocalStorageInit = () => {
+  if (localStorage.getItem('authToken') === null) {
+    localStorage.setItem('authToken', JSON.stringify(''))
+  }
+}
+
+export default LocalStorageInit
```

Comment:
On Monday Lene wrote something about separating the localStorage and that we should extract these functions in a folder called `lib`. Therefore, I would recommend to rename your folder according to her suggestions.